graphene_django/rest_framework/mutation.py

```diff
@@ -124,8 +126,13 @@ def __init_subclass_with_meta__(
     def get_serializer_kwargs(cls, root, info, **input):
         lookup_field = cls._meta.lookup_field
         model_class = cls._meta.model_class
-
         if model_class:
+            # check if input is Enum, and serialize value into input
+            for k, v in input.items():
+                attr = getattr(cls.Input, k)
+                type_ = getattr(attr, 'type')
+                if isinstance(v, Enum) and type_.get(v.value):
+                    input[k] = v.value
```

Comment:
Why isn't this new code just the following:

```python
            for k, v in input.items():
                if isinstance(v, Enum):
                    input[k] = v.value
```

i.e., what's the purpose of `type_.get(v.value)`? 

Related: would it be possible to use more verbose/explicit variable names so that it's clearer what this code means / is doing?

Reply:
You're right Thank you!

Reply:
I was a bit confused on where to use the `type_.get` I think i was supposed to pass it in for the test, I now have this, which also repros the issue on master.


In the test, now:

```
    kwargs = SomeCreateSerializerMutation.get_serializer_kwargs(
        None,
        mock_info(),
        **{
            "choice_type": SomeCreateSerializerMutation.Input.choice_type.type.get(
                "ASDF"
            )
        }
    )

``` 

Previous:

```
    kwargs = SomeCreateSerializerMutation.get_serializer_kwargs(
        None, mock_info(), **{"choice_type": MyFakeModelWithChoiceField.ChoiceType.ASDF}
    )

```

Reply:
updated the naming, and structure a bit to be more verbose and explicit in the code

I also removed the comment, I feel like now the code/naming/variables shows the intent more clearly, so it felt redundant to have the comment.